actors/verifreg/src/ext.rs

```diff
@@ -0,0 +1,43 @@
+use fvm_ipld_encoding::tuple::*;
+use fvm_ipld_encoding::tuple::{Deserialize_tuple, Serialize_tuple};
+use fvm_shared::address::Address;
+use fvm_shared::bigint::{bigint_ser, BigInt};
+
+pub mod datacap {
+    use super::*;
+
+    pub const TOKEN_PRECISION: u64 = 1_000_000_000_000_000_000;
+
+    #[repr(u64)]
+    pub enum Method {
+        // Non-standard.
+        Mint = 2,
+        Destroy = 3,
```

Comment:
This is my fault for not reviewing the last PR close enough but I don't understand why we are not using the standard Burn here.

Reply:
Copying over from slack:  It seems like the main difference is that some fields (operator, extra data) are moved from data to builtin actor code.  Since datacap logic is locked down to only accept from the verifreg operator it should be ok to send this data from the verifreg sender and ensure that it is exactly as expected: operator == verifreg addr, extra data is nil.  I think keeping the standard interface is simpler than removing the slight redundancy of passing fields that are implicit.

Reply:
We've resolved in person. This needs to bypass the usual restriction that would require the verifreg to be explicity authorised to burn tokens on behalf of a client. We can't allow the client to remove the verifreg's allowance.